common/test/common/ExperimentsTest.scala

```diff
@@ -0,0 +1,132 @@
+package experiments
+
+import conf.switches.Owner
+import org.joda.time.LocalDate
+import org.scalatest.{FlatSpec, Matchers}
+import test.TestRequest
+import ParticipationGroups._
+import play.api.mvc.RequestHeader
+
+class ExperimentsTest extends FlatSpec with Matchers {
```

Comment:
Can you add a test to check for duplicate participation groups? https://github.com/guardian/frontend/pull/8886/files#diff-09e577872825ce2f34f6dab651087231R13

Reply:
I thought about it but I wasn't not sure if we should absolutely prevent duplicate participation groups. There is nothing wrong using the same group in theory if you know what you are doing.
However I agree with you we can prevent it for now. It might actually never be an issue